app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -90,5 +91,10 @@ def how_to_verify_form_params
     rescue ActionController::ParameterMissing
       ActionController::Parameters.new(selection: [])
     end
+
+    def set_how_to_verify_presenter
+      @selfie_required = idv_session.selfie_check_required
+      @presenter = Idv::HowToVerifyPresenter.new(selfie_check_required: @selfie_required)
+    end
```

Comment:
@18F/identity-timnit I am conditionally rendering biometric specific text/ux elements on the How to Verify page. Can you confirm my understanding is correct;  `idv_session.selfie_check_required` is the value (and only value) I need to evaluate for biometrics.

Reply:
Hi @gina-yamada, that matches my understanding, too.  You can also see `idv_session.selfie_check_required` being used in the `idv/document_capture_controller` [here](https://github.com/18F/identity-idp/blob/main/app/controllers/idv/document_capture_controller.rb#L71), if it's helpful to have a reference.